pimd/pim_str.h

```diff
@@ -26,6 +26,9 @@
 
 #include <prefix.h>
 
+typedef struct in_addr pim_addr;
+#define PIM_PREFIX(p) ((p).u.prefix4)
```

Comment:
This needs a bit more thought, let's just put it separately.

(I haven't put this through my brain yet, it doesn't "look" great but maybe it's the best way to go.)

Reply:
Okay I have raised a separate draft PR for this https://github.com/FRRouting/frr/pull/10311